module/VuFind/src/VuFind/View/Helper/Root/RecordLink.php

```diff
@@ -244,6 +244,36 @@ public function getChildRecordSearchUrl($driver)
         return $escaper($url);
     }
 
+    /**
+     * Return search URL for all versions
+     *
+     * @param \VuFind\RecordDriver\AbstractBase $driver Record driver
+     *
+     * @return string
+     */
+    public function getVersionsSearchUrl($driver)
+    {
+        $mapFunc = function ($val) {
+            return addcslashes($val, '"');
+        };
+        $keys = $driver->tryMethod('getWorkKeys', [], []);
+        $imploded = implode('" OR "', array_map($mapFunc, $keys));
+        $urlParams = [
+            'join' => 'AND',
+            'lookfor0[]' => "\"$imploded\"",
+            'type0[]' => 'WorkKeys',
```

Comment:
I think this PR is missing a definition of WorkKeys in searchspecs.yaml; the "Show all versions (n) as search results" link results in an empty result set for me. I also wonder if there's a better way to approach this than creating an advanced search (since if you edit this advanced search, it won't fill in the form correctly unless you add WorkKeys to searches.ini as an advanced option...).

Reply:
Fixed searchspecs.yaml. I'm not sure I can think of a better way than an advanced search. I thought about doing it with filters, but then the filters don't have a filterspecs.yaml or similar, so they're pretty index-specific. What do you think about the approach I just added that makes sure the current handler is always displayed in the dropdown? That'd fix things also for e.g. saved searches containing handlers that were later disabled.

Reply:
Yes, this seems like a reasonable approach -- and that improvement to always display the active handler seems like something else we can merge ahead of the full PR, since it has other useful implications.